ecl/hthor/hthor.cpp

```diff
@@ -5017,6 +5110,7 @@ void CHThorJoinActivity::failLimit()
 
 const void *CHThorJoinActivity::nextRow()
 {
+    ActivityTimer t(activityStats, timeActivities);
```

Comment:
Why is an ActivityTimer instantiated for this method but not on others within this class? Actually, most related classes have ActivityTimers in their nextRow() methods but no other methods. Just checking to ensure that this is intended.